lib/ae_mdw/db/mutations/stats_mutation.ex

```diff
@@ -58,7 +59,9 @@ defmodule AeMdw.Db.StatsMutation do
         },
         state
       ) do
-    m_delta_stat = make_delta_stat(state, height, from_txi, next_txi, all_cached?)
+    Model.delta_stat(contracts_created: contracts_created) =
+      m_delta_stat = make_delta_stat(state, height, from_txi, next_txi, all_cached?)
```

Comment:
Instead of re-fetching something that has been just created, can we do `get(state, :contracts_created, 0),` ?

Reply:
I'm unsure what you mean, It's only being pattern matched on the response of make_delta_stat, which is called either way for other purposes

Reply:
you're right, we're not fetching it, we're just pattern-matching on a value that's being built